Signal/src/contact/OWSContactsManager.m

```diff
@@ -215,14 +215,10 @@ - (void)updateCachedDisplayNames
 {
     OWSAssert([NSThread isMainThread]);
 
-    // Preserve any existing values, so that contacts that have been removed
-    // from system contacts still show up properly in the app.
-    NSMutableDictionary<NSString *, NSString *> *cachedAccountNameMap
-        = (self.cachedAccountNameMap ? [self.cachedAccountNameMap mutableCopy] : [NSMutableDictionary new]);
-    NSMutableDictionary<NSString *, NSString *> *cachedFirstNameMap
-        = (self.cachedFirstNameMap ? [self.cachedFirstNameMap mutableCopy] : [NSMutableDictionary new]);
-    NSMutableDictionary<NSString *, NSString *> *cachedLastNameMap
-        = (self.cachedLastNameMap ? [self.cachedLastNameMap mutableCopy] : [NSMutableDictionary new]);
```

Comment:
As I played with these views - editing contacts, adding and removing numbers to contacts - it didn't make sense at all that we didn't update our display name cache to reflect these changes.  Happy to discuss, since I suspect our instincts here about the correct behavior may be different.